java/src/jmri/jmrix/lenz/XNetConsistManager.java

```diff
@@ -51,12 +52,12 @@ public boolean csConsistNeedsSeperateAddress() {
      * Add a new XNetConsist with the given address to consistTable/consistList.
      */
     @Override
-    public Consist addConsist(DccLocoAddress address) {
+    public Consist addConsist(LocoAddress address) {
         if (consistTable.containsKey(address)) { // no duplicates allowed.
             return consistTable.get(address);
         }
         XNetConsist consist;
-        consist = new XNetConsist(address, tc, systemMemo);
+        consist = new XNetConsist((DccLocoAddress)address, tc, systemMemo);
```

Comment:
Before this cast, you need to make sure the address is a DCCLocoAddress.  You should throw an illegal argument exception if it isn't.